vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Install and use packages on Linux
+
+## Description
+
+This tutorial guides you through setting up a new C++ "Hello World" project using the `fmt` library. You'll manage dependencies, build the application, and eventually run your project, all using vcpkg. This tutorial assumes you have vcpkg setup and ready to use. If not, check out our initial setup [tutorial](setup-vcpkg.md).
+
+### Prerequisites
+
+For this tutorial, you'll need: a terminal, `git`, `cmake`, and a C++ compiler.
+
+## Step 1: Setup the project
```

Comment:
Don't call out 'Step'. In general, this tutorial is sort of inventing a new format. We are trying to drive towards consistency in the Learn docs so that as people use multiple technologies, they don't have to learn new formats for how we present information. Please see our [cpub guide](https://review.learn.microsoft.com/en-us/help/patterns/level4/article-tutorial?branch=main#use-a-template) for how to write a tutorial. It also links to real-world examples. https://review.learn.microsoft.com/en-us/help/patterns/level4/article-tutorial?branch=main#use-a-template